src/ripple/basics/chrono.h

```diff
@@ -57,7 +57,10 @@ class NetClock
 };
 
 std::string to_string(NetClock::time_point tp);
-std::string to_string(std::chrono::system_clock::time_point tp);
+
+enum class Period { none, seconds, milliseconds, microseconds, nanoseconds };
+std::string to_string(std::chrono::system_clock::time_point tp,
+    Period period=Period::none);
```

Comment:
@HowardHinnant, it would be great if Mr. Chrono would weigh in on these changes to `to_string (chrono::time_point)`.  Thanks.